packages/ui/src/views/repo/repo-layout/index.tsx

```diff
@@ -31,34 +37,39 @@ export const RepoLayout = ({ useTranslationStore }: { useTranslationStore: () =>
-        <Tabs variant="navigation" value={activeTab}>
-          <TabsList>
-            <NavLink to={RepoTabsKeys.SUMMERY}>
-              <TabsTrigger value="summary">{t('views:repos.summary', 'Summary')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.CODE}>
-              <TabsTrigger value="code">{t('views:repos.files', 'Files')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.PIPELINES}>
-              <TabsTrigger value="pipelines">{t('views:repos.pipelines', 'Pipelines')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.COMMITS}>
-              <TabsTrigger value="commits">{t('views:repos.commits', 'Commits')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.PULLS}>
-              <TabsTrigger value="pulls">{t('views:repos.pull-requests', 'Pull Requests')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.BRANCHES}>
-              <TabsTrigger value="branches">{t('views:repos.branches', 'Branches')}</TabsTrigger>
-            </NavLink>
-            <NavLink to={RepoTabsKeys.SETTINGS}>
-              <TabsTrigger value="settings">{t('views:repos.settings', 'Settings')}</TabsTrigger>
-            </NavLink>
-          </TabsList>
-        </Tabs>
-      </SandboxLayout.SubHeader>
-      <Outlet />
+      <div className="bg-background-1 sticky top-0 z-40">
+        {breadcrumbs}
+        <SandboxLayout.SubHeader className="overflow-hidden h-[45px]">
+          <Tabs variant="navigation" value={activeTab}>
+            <TabsList>
+              <NavLink to={RepoTabsKeys.SUMMERY}>
+                <TabsTrigger value="summary">{t('views:repos.summary', 'Summary')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.CODE}>
+                <TabsTrigger value="code">{t('views:repos.files', 'Files')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.PIPELINES}>
+                <TabsTrigger value="pipelines">{t('views:repos.pipelines', 'Pipelines')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.COMMITS}>
+                <TabsTrigger value="commits">{t('views:repos.commits', 'Commits')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.PULLS}>
+                <TabsTrigger value="pulls">{t('views:repos.pull-requests', 'Pull Requests')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.BRANCHES}>
+                <TabsTrigger value="branches">{t('views:repos.branches', 'Branches')}</TabsTrigger>
+              </NavLink>
+              <NavLink to={RepoTabsKeys.SETTINGS}>
+                <TabsTrigger value="settings">{t('views:repos.settings', 'Settings')}</TabsTrigger>
+              </NavLink>
+            </TabsList>
+          </Tabs>
+        </SandboxLayout.SubHeader>
+      </div>
+      <div className="flex-1">
```

Comment:
i dont think you need this? lets try it out

Reply:
Removed